packages/astro/src/runtime/client/dev-overlay/plugins/audit/a11y.ts

```diff
@@ -358,7 +358,18 @@ export const a11y: AuditRuleWithSelector[] = [
 		match(element: HTMLElement) {
 			// innerText is used to ignore hidden text
 			const innerText = element.innerText.trim();
-			if (innerText === '') return true;
+			// Check for aria-label
+			const ariaLabel = element.getAttribute('aria-label')?.trim();
+			// Check for valid aria-labelledby
+			let ariaLabelledbyValid = false;
+			const ariaLabelledby = element.getAttribute('aria-labelledby')?.trim();
```

Comment:
Hmm, should this account for `aria-describedby` as well?

Reply:
I don't think so. I think the distinction is that it needs specifically an accessible name, and that accessible description and accessible name are separate things. This is also how deque axe-core tests things, which is what powers Lighthouse accessibility tests.


https://www.w3.org/TR/accname/#dfn-accessible-name